src/app/pages/Team.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  import TeamMemberProfile from "../components/TeamMemberProfile.svelte";
+
+  const coreMaintainerGithubUserNames: string[] = [];
+  const teamMemberGithubUserNames: string[] = ["ChrisChrisLoLo"];
+</script>
+
+<style>
+  .team-page {
+    width: 75%;
+    margin: auto;
+    margin-top: 1.5em;
+  }
+
+  .profile-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    justify-content: center;
+  }
+</style>
+
+<main>
+  <article class="team-page">
+    <nav style="text-align: center">
```

Comment:
I know this is copied from `Privacy.svelte` but we should avoid inline CSS like this 🙈